packages/features/bookings/lib/handleSeats/handleSeats.ts

```diff
@@ -41,9 +40,10 @@ const handleSeats = async (newSeatedBookingObject: NewSeatedBookingObject) => {
         {
           uid: rescheduleUid || reqBookingUid,
         },
+
         {
           eventTypeId: eventType.id,
-          startTime: evt.startTime,
+          startTime: new Date(evt.startTime),
```

Comment:
So that's what caused the double bookings, right? @joeauyeung

Reply:
Double bookings were caused when a `bookingId` wasn't present in the booking request. This would happen when a booking link was sent en mass and multiple people were booking at the same time. I created a test for this scenario in this PR.